src/compiler/core.ts

```diff
@@ -2631,5 +2631,9 @@ namespace ts {
         return (arg: T) => f(arg) && g(arg);
     }
 
+    export function or<T>(f: (arg: T) => boolean, g: (arg: T) => boolean) {

```

Comment:
`<||>` is much better name for this function bro

Reply:
If you can write that with emoji instead of reserved characters, it's doable. Like 🤷‍♂️ is kinda like that. Could use that. 😄